demos/python_demos/llm_text_generation/servable_stream/model.py

```diff
@@ -170,13 +170,16 @@ def execute(self, inputs: list):
             generate_kwargs["stopping_criteria"] = StoppingCriteriaList(stop_tokens)
 
         ov_model_exec = self.ov_model.clone()
+        self.token_count = 0
         def generate():
-            ov_model_exec.generate(**tokens, **generate_kwargs)
+            result = ov_model_exec.generate(**tokens, **generate_kwargs)
+            self.token_count += len([1 for x in result.numpy().flatten() if x not in tokenizer.convert_tokens_to_ids(tokenizer.all_special_tokens)])
 
         t1 = threading.Thread(target=generate)
         t1.start()
 
         for partial_result in streamer:
             yield serialize_completions(batch_size, partial_result)
-        yield [Tensor("end_signal", "".encode())]
+        t1.join()
+        yield [Tensor("end_signal", f"{str(self.token_count)}".encode())]
```

Comment:
Wouldn't it be better to return numpy array of uint32_t/uint64_t of size 1?
```
yield [Tensor(
  "end_signal",
  np.array([self.token_count], dtype=np.int])
)]
```

And then receive it on the client side:
```
num_of_tokens = result.as_numpy('end_signal')[0]
```